charts/pulsar/templates/pulsar_manager_cluster_initialize.yaml

```diff
@@ -0,0 +1,137 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if or .Release.IsInstall .Values.initialize }}
+{{- if or .Values.components.pulsar_manager .Values.extra.pulsar_manager }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.pulsar_manager.component }}-init"
+  namespace: {{ template "pulsar.namespace" . }}
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+    component: {{ .Values.pulsar_manager.component }}-init
+spec:
+  # This feature was previously behind a feature gate for several Kubernetes versions and will default to true in 1.23 and beyond
+  # https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+  {{- if .Values.job.ttl.enabled }}
```

Comment:
I wonder if it would make sense to explicit check for Kubernetes version and enable this by default on k8s 1.23+?
For example:
```
{{- if .Values.job.ttl.enabled or (semverCompare ">=1.23-0" .Capabilities.KubeVersion.Version) }}
  ttlSecondsAfterFinished: {{ .Values.job.ttl.secondsAfterFinished | default 600 }}
{
```

Reply:
Sure. I justed copied from the `pulsar-cluster-initalize.yaml`